frontend/src/pages/LogsPage/LogsTable/LogsTable.tsx

```diff
@@ -148,7 +150,9 @@ const LogsTableInner = ({
 }: LogsTableInnerProps) => {
 	const bodyRef = useRef<HTMLDivElement>(null)
 	const enableFetchMoreLogs =
-		!!moreLogs && !!clearMoreLogs && !!handleAdditionalLogsDateChange
+		(!!moreLogs || pollingExpired) &&
```

Comment:
Just to make sure I'm understanding this, we will show the button to refresh if polling has expired?

Reply:
See we are handing this case in `frontend/src/components/FeedResults/FeedResults.tsx`

Reply:
yes, and instead of showing "50+ more results" or nothing in case no more had been loaded, it will show "load more results"